lua/plenary/path.lua

```diff
@@ -15,8 +15,8 @@ local S_IF = {
   -- S_IFREG  = 0o100000  # regular file
   REG = 0x8000,
 }
-
 local path = {}
+path.home = vim.fn.expand('~')
```

Comment:
try `:lua print(vim.loop.os_homedir())`

Reply:
https://github.com/luvit/luv/blob/master/docs.md#uvos_homedir

Reply:
I can make that switch. Out of curiosity - what is vim.loop and how does it compare to vim.fn?

Reply:
`vim.loop` is basically that `uv` lib i send you the docs for. You can do `:help vim.loop`
`vim.fn` calls viml functions which makes things really slow while `uv` are bare `libuv` c bindings which makes them really fast.

Reply:
If you want an example how fast they are https://github.com/nvim-lua/plenary.nvim/pull/45
I implemented a `fd` in lua based on uv which can compare to `fd` in terms of speed. `fd` here run with `plenary.job` which uses `uv.spawn` (btw) so we have some overhead of collecting stdout, etc.